recipes/pony/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pony" %}
+{% set version = "0.7.1" %}
+{% set sha256 = "f3095910053ce60204e7af2dd5c3971bf4acf1db3c99ed7b7357cbfd7a9ac532" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - future  # [py27]
+
+test:
+  imports:
+    - pony
+    - pony.orm
+    - pony.utils
+
+about:
+  home: https://ponyorm.com
+  license: Apache-2.0
+  license_family: Apache
```

Comment:
Hi @dharhas, I am pretty sure the Apache 2 license, requires the `license_file` entry. If the license isn't in the pypi tarball, you can package it manually in the recipe: http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually